### PR TITLE
Add storage docs

### PR DESCRIPTION
**Pull request by JosepSampe**
Add storage docs
Fix pillow version
 
Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

File: config/README.md [hunk 1]
```diff
@@ -75,8 +75,10 @@ Storage backends:
 - [Ceph](storage/ceph.md)
 - [Redis](storage/redis.md)
 - [OpenStack Swift](storage/swift.md)
-- [Loclahost](storage/localhost.md)
```
Comment (gilv) [inline config/README.md:78]: @JosepSampe we don't need localhost anymore?

Comment (JosepSampe) [reply config/README.md:78]: ops! deleted by error

File: config/README.md [hunk 2]
```diff
@@ -75,6 +75,7 @@ Storage backends:
 - [Ceph](storage/ceph.md)
 - [Redis](storage/redis.md)
 - [OpenStack Swift](storage/swift.md)
+- [OpenStack Swift](storage/localhost.md)
```
Comment (gilv) [inline config/README.md:78]: @JosepSampe it's Localhost :) :)

**Comment by gilv** [comment]
@JosepSampe no more comments :) you can merge it...

**Pull request merged**